### ISSUE TITLE
vue.playstation.com

### ISSUE THREAD

**Issue by xxcriticxx** [jspenguin2017/uBlockProtector#685]
account needed
works only in usa

https://vue.playstation.com/watch/

![capture](https://user-images.githubusercontent.com/15007183/32463469-f4ff19c2-c30a-11e7-842b-eb1d4c324c9c.PNG)

**Comment by uBlock-user** [comment]
`playstation.com##script:inject(abort-on-property-read.js, adFileLoaded)`

Does that work ?

**Comment by jspenguin2017** [comment]
Well, I can't test then.

**Comment by uBlock-user** [comment]
yeah by @xxcriticxx can as he's able to reproduce the notice and I'm guessing he also has an account.

**Comment by xxcriticxx** [comment]
black screen page wont load 

![capture](https://user-images.githubusercontent.com/15007183/32465464-8c05786a-c311-11e7-9b99-579239962568.PNG)

**Comment by uBlock-user** [comment]
bait - https://vue.playstation.com/watch/assets/ads-f0af9057c104dbe6f53d35bb25304f8f.js
script - https://vue.playstation.com/watch/assets/baroness-31e1a2d8723cf2376c751cd80755c82b.js

**Comment by jspenguin2017** [comment]
If that didn't fix it then I'll need a test account.

**Comment by xxcriticxx** [comment]
i am going test this on few more devices but it looks fixed

**Comment by xxcriticxx** [comment]
please mark this as fixed!
